src/bridge/qos.go

```diff
@@ -4,13 +4,21 @@ import (
 	"errors"
 	"fmt"
 	log "minilog"
-	"strconv"
 )
 
 // Used to calulate burst rate for the token bucket filter
 const KERNEL_TIMER_FREQ uint64 = 250
 const MIN_BURST_SIZE uint64 = 2048
 
+type QosParams struct {
```

Comment:
Shouldn't these be float64s? If you were using the zero value string to indicate no change, you'll probably have to make them *float64s.